manifests/20-performance-profile.crd.yaml

```diff
@@ -161,6 +161,9 @@ spec:
                     highPowerConsumption:
                       description: HighPowerConsumption defines if the node should be configured in high power consumption mode. The flag will affect the power consumption but will improve the CPUs latency.
                       type: boolean
+                    perPodPowerManagement:
+                      description: perPodPowerManagement TODO
```

Comment:
Since this will hopefully be merged soon, you should probably fill in all the missing "TODO" text (there are several in the file).

Reply:
Sure

Reply:
Done